S/SimpleITK/build_tarballs.jl

```diff
@@ -0,0 +1,78 @@
+using BinaryBuilder, Pkg
+
+name = "SimpleITK"
+version = v"2.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://github.com/SimpleITK/SimpleITK/releases/download/v$(version)/SimpleITK-$(version).tar.gz", "b07bb98707556ebc2b79aac22dc14950749f509e5b43da8043233275aa55488a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mount -t tmpfs -o size=12G tmpfs /workspace/srcdir
+cd ..
+cd ..
+cd workspace/srcdir
+wget https://github.com/SimpleITK/SimpleITK/releases/download/v2.2.0/SimpleITK-2.2.0.tar.gz
+tar -xvzf ./SimpleITK-2.2.0.tar.gz 
+mkdir SimpleITK-build
+cd SimpleITK-build/
+cmake -DCMAKE_INSTALL_PREFIX:FILEPATH=/workspace/destdir -DCMAKE_BUILD_TYPE:STRING=RELEASE -DBUILD_SHARED_LIBS:BOOL=ON ../SimpleITK-2.2.0/SuperBuild
+make -j${nproc}
+rm -rf /workspace/srcdir/SimpleITK-build/ITK-build/lib/cmake
+cd SimpleITK-build
+make install
+cp /workspace/srcdir/SimpleITK-build/ITK-build/lib/* /workspace/destdir/lib
+cd /workspace/destdir/share
+mkdir licenses
+cd licenses
+mkdir SimpleITK
+cd SimpleITK
+cp /workspace/destdir/share/doc/SimpleITK-2.2/* ./ 
+cp /usr/lib/csl-glibc-x86_64/libgcc_s.so.1 /workspace/destdir/lib
+cp /usr/lib/csl-glibc-x86_64/libstdc++.so.6 /workspace/destdir/lib
+logout
```

Comment:
I was getting warnings for the same, so i included them, also do i not need to provide them since my package is built with gcc8, and in future when testing my code from a c++ interface, i need to provide access to these shared libraries to the LD_LIBRARY_PATH of my compiler?

Reply:
so is that not needed?

Reply:
I don't understand the problem you're trying to solve: if you're compiling ***everything*** from source you don't need to do any magic like that.

Reply:
ok thanks for that, its my first binarybuilder build so a bit messy.

Reply:
not even logout?